### ISSUE TITLE
Error in help documentation

### ISSUE THREAD

**Issue by jghiloni** [cloudfoundry/cli#1174]
## Command  

cf help disable-feature-flag

## What occurred

Output Text:

```
NAME:
   disable-feature-flag - Disable the use of a feature so that users have access to and can use the feature

USAGE:
   cf disable-feature-flag FEATURE_NAME

SEE ALSO:
   enable-feature-flag, feature-flags
```

## What you expected to occur
Output Text:
```
NAME:
   disable-feature-flag - Disable the use of a feature so that users have do not have access to and cannot use the feature

USAGE:
   cf disable-feature-flag FEATURE_NAME

SEE ALSO:
   enable-feature-flag, feature-flags
```
## CLI Version  

`cf version 6.28.0+9e024bdbd.2017-06-27`

## Platform & Shell Details  

Mac OS X / Linux

**Comment by dkoper** [comment]
Hi @jghiloni 

This command's description seems convoluted. I notice you're also struggling with the fix (`so that users have do not have`). :)
I'll ask my team to run it by a tech writer and fix it.
Thanks.

Regards,
Dies Koper
CF CLI PM

**Comment by jghiloni** [comment]
Yeah, that felt clunky to me too! Let's leave it to the professionals :)

**Comment by dkoper** [comment]
Addressed in cf CLI 6.29.0. Thanks again for reporting and taking a first stab at improving it.

Cheers,
Dies Koper
CF CLI PM
